### ISSUE TITLE
Running lvls on RasPi

### ISSUE THREAD

**Issue by riesentoaster** [riesentoaster/lvls#2]
> Next time I would like to try the setup on my raspy. Is there a possibility to write a bash script running on start up, so I don't need to connect display and keyboard on this small machine once it's installed?

Running a headless server posses a few issues in general:
- You have no status/logs, so it is very hard to know what is happening and what went wrong if something did.
- You can't control the server

With lvls in particular, there is also the issue that you could not read off the URLs of servers from the logs. You'd have to do a IP scan to get the IP and form the URLs yourself. If you are going through all the trouble of doing said scan, it is probably easiest to just SSH onto the pi and start the server like that.

One solution could be to set a static IP address to the Pi, which I wouldn't do with any device to reduce possible interference with possibly regardless unstable networks.

To sum up: 
- The easiest solution is to run lvls on your own computer or on one of the stream source computers. It runs anyway and could also serve as the server.
- If you want to run lvls on a RasPi, for the ease of use it might still be advisable to use a keyboard/screen.
- If you want to run your RasPi headless, I would connect to it by SSH and start the server from there. I could probably implement some process that would allow you to disconnect your session after starting the server, if necessary.
- If that's still too much hassle on site, you could do a IP scan, build your URLs yourself and start the server via a [cron job](https://www.adminschoice.com/crontab-quick-reference).

What do you think?
